eng/Versions.props

```diff
@@ -263,5 +263,7 @@
     <NodePackageVersion>$(runtimewinx64MicrosoftNETCoreRuntimeWasmNodeTransportPackageVersion)</NodePackageVersion>
     <!-- The package path for python in src/mono/mono.proj needs to be updated if this changes-->
     <EmsdkVersion>3.1.56</EmsdkVersion>
+    <!-- Microsoft.IO.Redist version to satisfy CG -->
+    <MicrosoftIORedistVersion>6.0.1</MicrosoftIORedistVersion>
```

Comment:
Does this file change mean this property will be updatable via darc?

Reply:
Is it available via darc? I didn't think it was. I just put it there to manage the version easier.

Reply:
I don't know either, that's why I asked :)

Reply:
I think this should be ok. If darc cannot update it because there are no new versions, then nothing should be affected.